src/Sentry/CaptureFeedbackResult.cs

```diff
@@ -0,0 +1,95 @@
+namespace Sentry;
+
+/// <summary>
+/// The result type of the <see cref="ISentryClient.CaptureFeedback"/> method
+/// </summary>
+public struct CaptureFeedbackResult
+{
+    /// <summary>
+    /// Creates a successful feedback capture result with the specified event Id.
+    /// </summary>
+    /// <param name="eventId"></param>
+    public CaptureFeedbackResult(SentryId eventId)
+    {
+        if (eventId == SentryId.Empty)
+        {
+            throw new ArgumentException("EventId cannot be empty", nameof(eventId));
+        }
+
+        EventId = eventId;
+    }
+
+    /// <summary>
+    /// Creates a failed feedback capture result with the specified error reason.
+    /// </summary>
+    /// <param name="errorReason"></param>
+    public CaptureFeedbackResult(CaptureFeedbackErrorReason errorReason)
+    {
+        EventId = SentryId.Empty;
+        ErrorReason = errorReason;
+    }
+
+    /// <summary>
+    /// The Id of the captured feedback, if successful. <see cref="SentryId.Empty"/> if feedback capture fails.
+    /// </summary>
+    public SentryId EventId;
+
+    /// <inheritdoc cref="CaptureFeedbackErrorReason"/>
+    public CaptureFeedbackErrorReason? ErrorReason;
```

Comment:
thought: `Nullable<T>` vs non-nullable

I am a bit hesitant about a `Nullable<enum>`.
I'm thinking about having an additional `0` value in the enum, like `Done` or `Succeeded`.
And calling the `enum` instead `CaptureFeedbackStatus`.
Similar to
- https://learn.microsoft.com/dotnet/api/system.buffers.operationstatus
- https://learn.microsoft.com/dotnet/api/system.threading.tasks.taskstatus